### ISSUE TITLE
Missbehaviour NetOffice.OutlookSecurity.Supress.Enabled

### ISSUE THREAD

**Issue by netoffice-bot** [NetOfficeFw/NetOffice#67]
<img src="https://avatars.githubusercontent.com/u/30236365?s=96" align="left" width="48" height="48" /> <strong>Issue by <a href="https://www.codeplex.com/site/users/view/Volarus" rel="author">Volarus</a></strong>
<em datetime="2013-03-13T09:23:07">Wed, 13 Mar 2013 09:23:07 GMT</em>
_Originally opened as https://netoffice.codeplex.com/workitem/19460_

----

for a detail description please see <a href="http://netoffice.codeplex.com/discussions/435265" rel="nofollow">http://netoffice.codeplex.com/discussions/435265</a>

tldr:
setting NetOffice.OutlookSecurity.Supress.Enabled = true 
<ul>
<li>not working if Outlook is working in the Background. Outlook icon in the task bar starts blinking. You have to bring it to the front, then the security dialog pops up for a short time and disapears. You get in big troubles because sometime you can't bring the dialog to the front</li>
<li>
as this function only tries to click a button, it tries this also in other Outlook dialogs, which makes working with Outlook impossible
</li>
</ul>
Thx in advance
